index.js

```diff
@@ -180,15 +180,17 @@ module.exports = {
             4,
             {
                 'CallExpression': {
-                    arguments: 'off'
+                    arguments: 1
```

Comment:
I think `arguments: 1` is how we format. To be exact though, I don't the function call expression exactly like in the example but rather like the following (presuming all of these are long enough to actually need breaking on multiple lines):

````javascript
foo(
    bar,
    baz,
    qux);
````

Reply:
Unfortunately this rule doesn't enforce the closing char to be at the end (both ways will work), only the parameter indentation. But I'll look closer to see if there is another rule to enforce that too.

Here is an example for this rule when fails (in master right now):
![image](https://user-images.githubusercontent.com/9974975/41726121-bebb0a1e-7571-11e8-811b-166646c59f91.png)
then the same fixed:
![image](https://user-images.githubusercontent.com/9974975/41726140-ce9adc20-7571-11e8-9737-e9501abeb7ae.png)

Reply:
I think I found the closing paren rule, but it needs a decision. When applied ```'function-paren-newline': [ 'error', 'never' ]```

These failures come up:
![image](https://user-images.githubusercontent.com/9974975/41840081-21af4a68-7865-11e8-929f-b8d6f0ba0f52.png)
Fixed:
![image](https://user-images.githubusercontent.com/9974975/41840102-2dc79404-7865-11e8-8198-2b0f93111be6.png)

![image](https://user-images.githubusercontent.com/9974975/41840193-6ae7d1b4-7865-11e8-9937-3391fba34336.png)
Fixed:
![image](https://user-images.githubusercontent.com/9974975/41840241-7f26db52-7865-11e8-8a31-7652c9751c03.png)

![image](https://user-images.githubusercontent.com/9974975/41840358-d75d71b4-7865-11e8-80c5-e676c614c66f.png)
Fixed:
![image](https://user-images.githubusercontent.com/9974975/41840372-e231c108-7865-11e8-87de-c5a29e49a6dc.png)

Not sure how this complies with our max line length rule though. What do you think?